### ISSUE TITLE
Trouble signing in with 32 character password with special characters

### ISSUE THREAD

**Issue by phiggins** [onlyhavecans/ReadItLater-Calibre-Plugin#20]
Hello,

I recently started using Calibre in Ubuntu to access Pocket. I use LastPass to generate passwords, and when I had signed up for Pocket I used my default settings: uppercase, lowercase, numbers, special characters, 32 characters long. An example might look like this:

9O!e%XGOfBAeuhQ!m4NcoQHwT8^#I0X^

When attempting to use Pocket as a news source in Calibre, I would get an error, and when viewing the details it was an HTTP 401 error from the Pocket server complaining of a username/password mismatch. I had previously had no trouble accessing Pocket from the web or via the Android app. I attempted multiple times to enter my password correctly in Calibre, going as far as manually verifying character by character that it was the same as in LastPass. When this still didn't work, I went to the Pocket website and changed my password to one that was 20 characters long and did not contain special characters. Upon entering this new password in Calibre, using Pocket as a news source began to work.

I am new to using Calibre, and it was not obvious how to determine what version of Calibre I am using.

**Comment by kaushikvira** [comment]
Yup.. iI ran into same issue. if password has some # ran into same issue.
